### PR TITLE
if not autoresolved, display a more suitable message

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: pages/user/update/index.js [hunk 1]
```diff
@@ -44,13 +44,16 @@ module.exports = settings => {
       }
     };
     return req.api(`/me`, opts)
-      .then(() => next())
+      .then(response => {
+        const status = get(response, 'json.data.json.data.status');
```
Comment (lennym) [inline pages/user/update/index.js]: WUT?! Is that for real? We may want to... not.

Comment (joefitter) [reply pages/user/update/index.js]: yeah, it's whack.

Comment (joefitter) [reply pages/user/update/index.js]: json data this, json data that

Comment (lennym) [reply pages/user/update/index.js]: How much do we want to resolve that now vs _the bucket_?

Comment (joefitter) [reply pages/user/update/index.js]: it _works_ 😬

Comment (joefitter) [reply pages/user/update/index.js]: but yeah I 100% agree it's completely mental. I don't think this is the only place it's used though so tempted to say let's add a bug

Comment (lennym) [reply pages/user/update/index.js]: It's a couple of lines in the workflow client in asl-service, so I'm going to fix it. I can't handle it being the way it is and it's easier to fix now before it spreads to other places.

Comment (lennym) [reply pages/user/update/index.js]: Turns out it isn't asl-service, it's specific to this one endpoint 

https://github.com/UKHomeOffice/asl-public-api/pull/126

**Comment by lennym** [comment]
I've made the change to un-💩-ify the api response, so we can unjsondatajsondataify here now.

**Comment by joefitter** [comment]
I think you mean dejsondatajsondataify

**Pull request merged**